hazelcast-client/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -527,7 +529,7 @@ private void handleAt(String first) {
     }
 
     private void handleEcho(String command) {
-        if (!Thread.currentThread().getName().toLowerCase().contains("main")) {
+        if (!Thread.currentThread().getName().toLowerCase(StringUtil.LOCALE_INTERNAL).contains("main")) {
```

Comment:
I think this one is called a train wreck. But wouldn't it anyway be easier to use the `StringUtil` methods (with static imports)?
```
String threadName = lowerCaseInternal(Thread.currentThread().getName());
if (!threadName.contains("main")) {
   ...
}
```
Same comment applies to all other code changes.

Reply:
Using the `toLowerCase()` comes more readable to me.
And the constant `LOCALE_INTERNAL` is a public API, so I wouldn't hesitate to use it.